chain/tron/client.go

```diff
@@ -169,15 +96,80 @@ func (client *Client) FetchLegacyTxInput(ctx context.Context, from xc.Address, t
 }
 
 // SubmitTx submits a Tron tx
+// Submission of unstake/stake/withdraw transactions relies on proper handling
+// of "ErrRequiresResubmission" error. It's modeled this way to avoid blocking and match
+// treasury behavior.
 func (client *Client) SubmitTx(ctx context.Context, tx xc.Tx) error {
-	bz, err := tx.Serialize()
-	if err != nil {
+	// Submit legacy transactions that doesn't contain any metadata
```

Comment:
I wouldn't call it legacy, as having this continue to work for a standard serialization of TRON transaction is definitely useful still.